app/src/main/java/com/example/next_show/data/RecommendationClient.java

```diff
@@ -43,11 +43,10 @@ public RecommendationClient(Context context, User currentUser) {
     }
 
     public void fetchRelatedShows(Shows traktShows, ShowAdapter adapter) {
-        // use given show ID to generate related, recommended shows -> randomize it!
+        // use randomly picked show ID to generate related, recommended shows
```

Comment:
Ah makes sense -- maybe you should save the random val into a variable here then:

```
final int randomShowIndex = new Random().nextInt(savedShows.size());
```